doc/man3/X509_get_subject_name.pod

```diff
@@ -25,16 +34,29 @@ subject names
 
 =head1 DESCRIPTION
 
-X509_get_subject_name() returns the subject name of certificate B<x>. The
+X509_NAME_hash_ex() returns the SHA1 hash value of name I<x> or 0 on failure,
```

Comment:
One more nit - as the value is not a full SHA1 hash, it should be mentioned here (or maybe even better in RETURN VALUES below).  Something like - `returns the first four bytes of SHA1 hash value of name I<x>. The value is converted to unsigned long in little endian order`.

Reply:
Indeed, good point! 
Done in the RETURN VALUES section, and abstracted the description for `X509_NAME_hash_ex()`, `X509_NAME_hash()`, and `X509_subject_name_hash` accordingly.